release-team/role-handbooks/release-notes/README.md

```diff
@@ -109,16 +109,15 @@ The main task of the Release Notes team is the generation of the release notes d
 
 At least one member of the Release Notes Team should be responsible for [setting up](https://github.com/kubernetes/release/tree/master/docs/krel#installation) and [running](https://github.com/kubernetes/release/blob/master/docs/krel/release-notes.md) the release-notes subcommand of krel to generate both versions of the release notes after each Patch Release:
 
-1. Update the release notes draft, a markdown file which will become the final document which will encompass all release notes written by contributors during the current release cycle. See previous drafts for versions [v1.18](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.18/release-notes-draft.md), [v1.17](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.17/release-notes-draft.md) or [v1.16](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.16/release-notes-draft.md).
+1. Update the release notes draft, a markdown file which will become the final document which will encompass all release notes written by contributors during the current release cycle. See previous drafts for versions [v1.20](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.20/release-notes-draft.md), [v1.19](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.18/release-notes-draft.md) or [v1.18](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.19/release-notes-draft.md).
 
-2. The team is responsible for adding Patch Release (e.g. 1.15.1, etc.) notes to the release notes website at https://relnotes.k8s.io. The krel release notes subcommand will automatically generate the necessary JSON files and patch the release notes website source.
+2. The team is responsible for adding Patch Release (e.g. [1.20.1](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md#v1201), etc.) notes to the release notes website at https://relnotes.k8s.io. The krel release notes subcommand will automatically generate the necessary JSON files and patch the release notes website source.
```

Comment:
Yeah I think we're still responsible for this for now, this should go away sometime this year 😄

Reply:
Would a note be a good option for this? We can add a note it will be outdated by 2022.

Reply:
hmm I think I prefer something like "there is a work in progress to make this effort more integrated" and link it to https://github.com/kubernetes/release/issues/1087